components/Drawer.tsx

```diff
@@ -0,0 +1,99 @@
+import Drawer from '@mui/material/Drawer'
+import List from '@mui/material/List'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import { Box } from '@mui/system'
+import StarIcon from '@mui/icons-material/Star'
+import Image from 'next/image'
+import Collapse from '@mui/material/Collapse'
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+import React from 'react'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import SettingsIcon from '@mui/icons-material/Settings'
+
+export default function NavigationDrawer() {
```

Comment:
have this component take in a prop called `open` that controls the state of the drawer

Reply:
Just to be clear, are you saying I just need to add that prop for future use or actually implement a solution like on MUI's docs https://mui.com/material-ui/react-drawer/ -> Responsive Drawer

Reply:
@collins-self for now just add the prop, im going to update the layout story to wire it all together.